openmetadata-ui/src/main/resources/ui/src/assets/svg/ic-share.svg

```diff
@@ -1,6 +1,6 @@
-<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
as we are removing width and height, please verify the places where this svg is used. and if needed provide W/H from there.

Reply:
yes, it is verified and none is breaking!